src/components/LocationPage/ChartsHolder.tsx

```diff
@@ -145,74 +144,63 @@ const ChartsHolder = ({ projections, region, chartId }: ChartsHolderProps) => {
   // TODO(pablo): Create separate refs for signup and share
   return (
     <>
-      {!projection ? (
-        <NoCountyDetail
-          countyId={
-            region.regionType === RegionType.COUNTY ? region.urlSegment : ''
-          }
+      <ChartContentWrapper>
```

Comment:
Consider removing other conditionals on projection above https://github.com/covid-projections/covid-projections/pull/2170/files#diff-074b0a67b501ec0a6b4ea0047e8af2f238c0e7ffb9b70533ed0ba791d6ccedf9R104

Reply:
ooo good call, will add that in my next PR (that's coming pretty much immediately :) )